src/edit.js

```diff
@@ -26,20 +27,67 @@ import Block from './block';
  * @return {WPElement} Element to render.
  */
 export default function Edit( { attributes, setAttributes } ) {
-	const { freeShippingFrom } = attributes;
+	const {
+		insufficientTotals,
+		sufficientTotals,
+		freeShippingFrom,
+	} = attributes;
 
 	return (
 		<div { ...useBlockProps() }>
 			<InspectorControls>
-				<PanelBody title="Settings" initialOpen={ true }>
+				<PanelBody
+					title={ __( 'Settings', 'free-shipping-progress-bar' ) }
+					initialOpen={ true }
+				>
+					<TextareaControl
+						label={ __(
+							'Insufficient totals message',
+							'free-shipping-progress-bar'
+						) }
+						help={ __(
+							'Provide the message for insufficient totals. %s is a placeholder for the calculated remaining value and should not be removed.',
```

Comment:
Consider using something more obvious than %s. Consider showing a notice if %s isn't present in the text. We do this in the Terms and Conditions block in Checkout, in which we show an error if the text doesn't contain a string, and offer a reset.

Reply:
Thanks for pointing out the example in the terms and conditions block within the checkout block.

Reply:
@senadir Given that the string can occur in various languages, I'm only showing a warning, but I haven't implemented the action to copy the original string `Spend only {REMAINING_AMOUNT} more to get free US shipping!` back into the `<PlainText>` field. I'm keen to hear your thoughts on this.